student.rb

```diff
@@ -0,0 +1,13 @@
+# student.rb
+require_relative 'person'
+
+class Student < Person
+  def initialize(id, classroom)
+    super(id)
+    @classroom = classroom
+  end
```

Comment:
- [ ] There should be default parameters for age, name, and parent_permission but you are yet to add the parameter and the default data.